deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -102,43 +127,83 @@ def run_logic(self, context: Context) -> CheckResult:
             value: drift score.
             display: label distribution graph, comparing the train and test distributions.
         """
+        if (self.probability_drift == 'true') and (context.task_type == TaskType.REGRESSION):
+            raise DeepchecksValueError('probability_drift="true" is not supported for regression tasks')
+
         train_dataset = context.train
         test_dataset = context.test
         model = context.model
 
-        train_prediction = np.array(model.predict(train_dataset.features_columns))
-        test_prediction = np.array(model.predict(test_dataset.features_columns))
-
-        drift_score, method, display = calc_drift_and_plot(
-            train_column=pd.Series(train_prediction.flatten()),
-            test_column=pd.Series(test_prediction.flatten()),
-            value_name='model predictions',
-            column_type='categorical' if train_dataset.label_type != TaskType.REGRESSION else 'numerical',
-            margin_quantile_filter=self.margin_quantile_filter,
-            max_num_categories_for_drift=self.max_num_categories_for_drift,
-            max_num_categories_for_display=self.max_num_categories_for_display,
-            show_categories_by=self.show_categories_by,
-            categorical_drift_method=self.categorical_drift_method,
-            with_display=context.with_display,
-        )
+        # Flag for computing drift on the probabilities rather than the predicted labels
+        proba_drift = ((context.task_type == TaskType.MULTICLASS) and (self.probability_drift == 'auto')) or \
```

Comment:
binary and self.probability_drift == 'true'?

Reply:
didn't follow... If it's set to 'true', then use proba, period. 
(I earlier test if it's 'true' and regression, in which case I raise an exception)

Reply:
yep you are right, misread where the ( were placed